docs/rules/sort-imports.md

```diff
@@ -136,6 +138,34 @@ import c from 'baz.js';
 
 Default is `false`.
 
+### `ignoreDeclarationSort`
+
+Ignores the sorting of import declaration statements.
+
+Examples of **incorrect** code for this rule with the default `{ "ignoreDeclarationSort": false }` option:
+
+```js
+/*eslint sort-imports: ["error", { "ignoreDeclarationSort": false }]*/
+import 'foo.js'
+import 'bar.js'
```

Comment:
This examples of incorrect code wasn't warned: https://eslint.org/demo/#eyJ0ZXh0IjoiLyplc2xpbnQgc29ydC1pbXBvcnRzOiBbXCJlcnJvclwiLCB7IFwiaWdub3JlRGVjbGFyYXRpb25Tb3J0XCI6IGZhbHNlIH1dKi9cclxuaW1wb3J0ICdmb28uanMnXHJcbmltcG9ydCAnYmFyLmpzJ1xyXG4iLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjgsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=

Is this intentional?